parm/archive/gdasocean.yaml.j2

```diff
@@ -2,8 +2,12 @@ gdasocean:
     {% set head = "gdas.ocean.t" + cycle_HH + "z." %}
     name: "GDASOCEAN"
     target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gdasocean.tar"
+    # the FHMIN-hour forecast is skipped when DOIAU="NO"
     required:
-        {% for fhr in range(FHMIN, FHMAX + 1, FHOUT_OCN) %}
+        {% if DOIAU %}
+        - "{{ COMIN_OCEAN_HISTORY | relpath(ROTDIR) }}/{{ head }}inst.f{{ '%03d' % FHMIN }}.nc"
```

Comment:
I don't quite know what needs to be saved; did the behavior here changed for the IAU case? (I see there was a loop, and now it's just a single forecast).

Reply:
The the logic of the IAU case is unchanged, the non-IAU case lacks (in the current form) the f000 forecast.

Reply:
So on develop `for fhr in range(FHMIN, FHMAX + 1, FHOUT_OCN)` would only generate `FHMIN` in the loop?

Reply:
@shlyaeva The `for` loop is still present below the new `if` block, though it now starts at `FHMIN + FHOUT`.  If `DOIAU`, then `FHMIN` is also archived, so all of the same files end up being archived.  It only changes when `DOIAU=NO`.